packages/pylimer-tools/test_pylimer_tools.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from pytest_pyodide import run_in_pyodide
+
+@run_in_pyodide(packages=[
+  "pylimer-tools", "numpy", "micropip"
+])
+async def test_pylimer_tools_cpp(selenium):
+    import micropip
+    await micropip.install("pint")
```

Comment:
I don't quite like that this test requires installing another package, as it would make network requests every time we need to run this test.

Would there be a way to test it without installing an additional package?

Reply:
Done in 37dbe98bb915a1d52743e00a58a18a0449ed02f8